Game/Resources/Constant Scene/Minecart/Minecart.json

```diff
@@ -385,25 +385,26 @@
 							},
 							"Shape" : 
 							{
-								"sphere" : 
+								"cylinder" : 
 								{
-									"radius" : 0.95000004768371582
+									"length" : 1.3999999761581421,
+									"radius" : 0.059999998658895493
 								}
 							},
 							"Trigger" : 
 							{
 								"trigger" : "A Trigger",
 								"triggerActive" : false,
 								"triggerCharges" : 0,
-								"triggerCollidedEntity" : 1508919751,
+								"triggerCollidedEntityUID" : 1508919751,
```

Comment:
This property was only used briefly after the trigger editor removed the dropdown in favor of an input box and was subsequently removed in #874. The line that was removed is indeed correct because otherwise the property will not be found when creating the trigger component from JSON and as such it will use the default of 0 instead, causing it to not detect collisions with the entity.